openedx/core/lib/xblock_builtin/xblock_discussion/static/js/mathjax_include.js

```diff
@@ -0,0 +1,35 @@
+var vendorScript;
+if (typeof MathJax === 'undefined') {
+    vendorScript = document.createElement('script');
+    vendorScript.onload = function() {
+        'use strict';
+        var MathJax, setMathJaxDisplayDivSettings;
```

Comment:
@pomegranited Doh... nice catch. That's what happens when it's late and you're just trying to make the JS linter happy ;)